### PR TITLE
DOCSP 30174 clarifying estimatedCopiedByes and estimatedTotalBytes

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

clarifying that estimatedCopiedByes and estimatedTotalBytes are approximated before migration and should not be used as an indicator of progress. 

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-30174/reference/api/progress/#successful-response

## JIRA

https://jira.mongodb.org/browse/DOCSP-30174

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65d7783e7111165982411eb5

## SELF-REVIEW CHECKLIST

- [x] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jocelyn-mendez1 — APPROVED** [review]
Thank you @ltran-mdb2! This LGTM!

File: source/includes/api/tables/progress-response.rst [hunk 1]
```diff
@@ -78,6 +87,15 @@
        and divide the result by the value of the ``estimatedTotalBytes`` field
        . Then, multiply the result by 100.
 
+       .. note::
+
+         ``mongosync`` approximates the estimated number of copied bytes
+         prior to migration and does not update this value during the
+         synchronization process. This value does not reflect changes
+         made to the source cluster during sync and is not an accurate
+         indicator of migration progress.
```
Comment (suspicious-mango) [inline source/includes/api/tables/progress-response.rst:96]: This should say something like:

mongosync only reflects the bytes copied by the current instance of mongosync. The value does not reflect bytes copied by other instances of mongosync and can become larger than estimatedTotalBytes due to retried operations. This value is not an accurate indicator of migration progress.

**Review by suspicious-mango — CHANGES_REQUESTED** [review]
correction to estimatedCopiedBytes

File: source/includes/api/tables/progress-response.rst (hunk 1 above)
Comment (suspicious-mango) [reply source/includes/api/tables/progress-response.rst:99]: Ahh apologies, I didn't see that the top part of the description for `estimatedCopiedBytes`. Then there is no need to restate "This value only reflects the bytes copied by the current instance of mongosync".

But the following section is incorrect and should be removed. mongosync does in fact compute the estimated number of copied bytes during the migration by adding up the size of documents that it copies. This value will reflect changes made to the source cluster during sync and can be higher than ``estimatedTotalBytes`` if documents are added to the source or lower if documents are deleted from the source.
```
         mongosync approximates the estimated number of copied bytes
         prior to migration and does not update this value during the
         synchronization process. This value does not reflect changes
         made to the source cluster during sync and is not an accurate
         indicator of migration progress.
```

Comment (ltran-mdb2) [reply source/includes/api/tables/progress-response.rst]: no worries, just made your suggested change!

**Review by suspicious-mango — APPROVED** [review]
LGTM.

**Pull request merged**